### ISSUE TITLE
-col suffix makes meshes not export / load in godot.

### ISSUE THREAD

**Issue by TheConceptBoy** [godotengine/godot-blender-exporter#326]
**OS:**
Win 7 Latest update

**Godot version:**
3.2

**Blender version:**
2.81a

**Issue description:**
Here is a simple blender scene. 2 objects, both need collisions, both have the -col suffix. One of the objects does not get loaded in godot. If I remove the -col from both objects, both get loaded in godot just fine but that means no collisions are made and I have to create them manually.

**Minimal reproduction project:**
[pillar_not_exporting.zip](https://github.com/godotengine/godot-blender-exporter/files/4227184/pillar_not_exporting.zip)

**Comment by Shatur95** [comment]
Can confirm. It works as expected only in first import. If I press re-import, the model becomes broken.

**Comment by slapin** [comment]
This is Godot importer bug, not exporter bug.
I have to remove .import directory frequently. The issue looks like sometimes during reimport data are loaded from cache instead of loading directly from files and everything gets messed up. Now I got used to remove .import directory every time after importing complex scenes.

**Comment by Shatur95** [comment]
@slapin, should I send this issue to Godot instead? Re-import is broken even if the model not changed. Some parts of the model just disappears.

**Comment by slapin** [comment]
You should be very careful creating the report on this issues, as all mine
were closed and
nobody could reproduce anything. I debugged that heavily but I could not
get to the root cause.
I think at this stage without proper fix together with issue created, due
to lack of testing on 3D
side and actual people who actually do something 3D and presence of people
who troll bug
reports, nobody will believe you.
The problem occurs only in case when some object references another object
which is updated,
so cached entry is in presence. It seems that people won't set things like
that and the setting up
of such project (or use yours) will be too much effort for them.

**Comment by Shatur95** [comment]
But this bug is easy to reproduce. I just not sure that this is a Godot issue. For example, this issue do not exists with GLTF.

**Comment by slapin** [comment]
I was not able to find anybody else to reproduce this bug, people said it was not reproducible for them.

**Comment by Shatur95** [comment]
I think that they could not reproduce them because they used another exporter. And I still think that this is a problem with this plugin, because other formats just works.

**Comment by slapin** [comment]
The problem might be with importer part, which is part of Godot. Not the
exporter.

**Comment by slapin** [comment]
Also are you sure you can't reproduce the problem with other exporters? Try making imported scene a child of another scene, open the parent scene and press reimport button. The other way is create inherited scene from imported one, save it then open it and press reimport button on imported one.

**Comment by Shatur95** [comment]
@slapin, Yes, I can't reproduce it with glTF, for example. what I tested:
1. Export the following [model](https://github.com/Zoriya/asset-sources/blob/master/Environment/Landscape/Island.blend) with the default settings as glTF.
2. Copy the exported file to Godot project folder.
3. Open `.gltf` file.
4. Select "Create inherited scene".
5. Save it.
6. Press "Re-import" on the "Import" tab.
And everything works correctly. Now I currently using this format for my [small project](https://github.com/Zoriya/zoriya).

If I export the model above with this plugin I will have some missing parts after re-import. It happens even without creating an inherited scene.

OS: Arch Linux
Blender: 2.82
Godot: 3.2.1

**Comment by rainerdeyke** [comment]
Here's how to make this reproducible:

- Export a scene.
- Save the .escn file (in another directory).
- Export the scene again.
- Save the (new) .escn file (in the other directory, under a different name).

The bug should now be reproduced without using an exporter:
- Copy the second .escn into a Godot project directory.
- Confirm that this .escn is not, in itself, broken by opening it in Godot.
- Delete this .escn file from the Godot project directory again.
- Copy the first .escn file into a Godot project directory.
- Open it.
- Copy the second .escn file over the first one.
- Watch how the scene in Godot is now broken.
